influxdb_client/client/flux_csv_parser.py

```diff
@@ -230,4 +233,12 @@ def add_column_names_and_tags(table, csv):
 
     def _insert_table(self, table, table_index):
         if self._serialization_mode is FluxSerializationMode.tables:
-            self.tables.insert(table_index, table)
\ No newline at end of file
+            self.tables.insert(table_index, table)
+
+
+def get_date_parse_function():
```

Comment:
Please, move the function into `date_utils.py` and cache the result there => `ModuleNotFoundError` should be raised only once.